src/main/java/heat/db/DBHelper.java

```diff
@@ -0,0 +1,50 @@
+package heat.db;
+
+import heat.config.DatabaseConfig;
+import heat.utils.ResourceExtractor;
+
+import java.sql.*;
+
+public class DBHelper {
+
+    public static void createUser(User user) {
+        try {
+            final String INSERT_USER = ResourceExtractor.getSql(ResourceExtractor.SQL_INSERT_USER);
+            Connection connection = DatabaseConfig.getDataSource().getConnection();
```

Comment:
Not sure about this, but connection can time out. We need to investigate this case. Maybe it worth to check in the DatabaseConfig class if the connection is still opened.

Reply:
I think, that getConnection() method can do it without me.

Reply:
I have some bad experience about `getConnection`, that's why I'm asking :)

Reply:
+++, I also had problems with it.

Reply:
Agreed that it can be handled by the data source object. Let's test in production :)